RELEASES.md

```diff
@@ -97,6 +97,21 @@ As rule of thumb, the following changes will **NOT** be automatically accepted i
 * **API-breaking changes**, i.e. changes that prevent client applications to *build without modifications* to the client application's source code.
 * **CLI-breaking changes**, i.e. changes that require usage changes for CLI users.
 
+## Version matrix
```

Comment:
I added it to RELEASES.md, but open to suggestions to have it another place.

We can add later on a link to the matrix on the README.md when working on #1747.

Reply:
sounds good for now, we can always iterate if it needs moving later :)